cilium/cilium.io
# feat: update ama card

**This PR:**
- Fixed a problem with text padding in subscribe form on the Get involved page;
- Changed the photo in the ama card to a bee;
- Added cta card at the bottom of the event page;

**Steps to test**

- Open the [Events page](https://deploy-preview-314--cilium.netlify.app/events/)
- Open the [Get involved page](https://deploy-preview-314--cilium.netlify.app/get-involved/)
- Confirm that everything looks and works as expected
